samples/hello/src/com/microsoft/aad/adal/hello/MainActivity.java

```diff
@@ -281,12 +282,17 @@ public void onClickUseToken(View view) {
     }
 
     public void onClickClearTokens(View view) {
-        if (mAuthContext != null && mAuthContext.getCache() != null) {
-            displayMessage("Clearing tokens");
-            mAuthContext.getCache().removeAll();
-        } else {
-            textView1.setText("Cache is null");
-        }
+        try {
+			if (mAuthContext != null && mAuthContext.getCache() != null) {
+			    displayMessage("Clearing tokens");
+			    mAuthContext.getCache().removeAll();
+			} else {
+			    textView1.setText("Cache is null");
+			}
+		} catch (final UsageAuthenticationException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
@heidijinxujia Does this actually log?  Why are you throwing the exception away.  Exactly which method can throw this and under what circumstances?